### ISSUE TITLE
multiple Databases

### ISSUE THREAD

**Issue by ghost** [firecat53/keepmenu#48]
Is it possible to have multiple databases loaded parallel? I always have to kill the deamon and load the other DB, it would be nice to just be able to search them both. Is that possible and I just missed how?

**Comment by pkfm** [comment]
From `config.ini.example`:

```
# database_1 = <path/to/database> ~ for $HOME is ok
# keyfile_1 = <path/to/keyfile>
# password_1 = database password  **INSECURE**
# password_cmd_2 = <command to generate db password>
# database_2 = <path/to/second database>
# etc....
```

Note the `database_2` entry. From this it seems you can load an arbitrary number of databases. Also I believe tools exist to merge kdb(x) databases.

**Comment by firecat53** [comment]
No, you can have multiple databases defined, but only one can be opened at a time. It could become possible with some work...PR's welcome!

**Comment by firecat53** [comment]
Closed by #72.
